src/main/java/movie/tickets/model/dto/UserRequestDto.java

```diff
@@ -1,8 +1,22 @@
 package movie.tickets.model.dto;
 
+import javax.validation.constraints.NotNull;
+import javax.validation.constraints.Size;
+import movie.tickets.annotation.EmailValidation;
+import movie.tickets.annotation.FieldsValueMatch;
+
+@FieldsValueMatch(
+        field = "password",
```

Comment:
If you will use dto instead of object you will no need fields.

Reply:
it will look better, you can try if you want